docs/ContribOperators.md

```diff
@@ -2195,6 +2196,33 @@ No versioning maintained for experimental ops.
 </dl>
 
 
+### <sub>experimental</sub> <a name="com.microsoft.Inverse"></a><a name="com.microsoft.inverse">**com.microsoft.Inverse**</a>
```

Comment:
Inverse is now an official ONNX op. I assume people should use that instead of an experimental contrib op, so do we need to add documentation here for it?

https://github.com/onnx/onnx/blob/master/docs/Operators.md#Inverse

Reply:
Inverse should be removed from ONNX. At least that was the last decision. cc @codemzs

Reply:
@pranavsharma is correct, Inverse is being removed https://github.com/onnx/onnx/pull/2747

Reply:
Please sync to latest. Inverse is removed.

Reply:
@skottmckay, @pranavsharma, @codezms It looks like Inverse is still being registered as a contrib operator the code (https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/contrib_ops/cpu/inverse.cc#L24) and this is (AFAIK) how these docs get generated. So is the code correct? And the docs?